mod-director-core/src/main/java/net/jan/moddirector/core/configuration/type/UrlRemoteMod.java

```diff
@@ -131,4 +138,14 @@ public RemoteModInformation queryInformation() {
             return new RemoteModInformation(name, name);
         }
     }
+
+	@Override
+	public String folderName() {
+		return this.folderName;
+	}
+	
+	@Override
+	public int forceInject() {
```

Comment:
Instead of adding these here, add them into `ModDirectorRemoteMod`, as they apply to all mod types.